pkg/webhook/pod/tikv_creater.go

```diff
@@ -69,6 +69,7 @@ func admitCreateTiKVPod(pod *core.Pod, pdClient pdapi.PDClient) *admission.Admis
 	// if the pod which is going to be created already have a store and was in evictLeaderSchedulers,
 	// we should end this evict leader
 	for _, store := range stores.Stores {
+		// This is incluster operation, so select the store by podName is acceptable when multi-cluster feature is enabled. The stores are not including peerStores.
```

Comment:
The stores are retrieved from the PD API in L37, so it includes all stores in all clusters.